brendel-group/mission_db
# Rename missionlister folder to backend

I did rename the folder `missionlister` to `backend` because it is more intuitive.\
This equals to renaming a django project.\
I also renamed the folder `missionlister/missionlister` to `backend/backend` because this folder was created by django and should have the same name as the django project.\
To do this I had to change every occurency of `missionlister` in any file to `backend`.

## How to test
To confirm, that this didn't break anything you can run the django tests with:
```bash
python3 manage.py test
```
And because this also affects the CLI you have to manually test the CLI, because it is not covered by the django tests. Refer to the [CLI documentation](https://github.com/brendel-group/polybot_mission_db/blob/feature/rename-django-project/docs/cli/README.md) for testing.\
You can also test the REST API manually please refer to the [REST API documentation](https://github.com/brendel-group/polybot_mission_db/blob/feature/rename-django-project/docs/restapi/README.md) for this.
